docs/source/modin/supported/series_supported.rst

```diff
@@ -296,6 +296,8 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``pipe``                    | N                               |                                  |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
+| ``plot``                    | D                               |                                  |                                                    |
```

Comment:
same here, I'd also maybe add to best practices a section that says:

If you want to plot large amounts of data, consider first minimizing the data being plotted. I.e., precompute aggregates/groupbys before plotting together with an example.

Reply:
I think users are smart enough to know this. I actually think they know this better than we do ;)